skytemple_files/common/rw_value.py

```diff
@@ -18,6 +18,7 @@
 from abc import ABC, abstractmethod
 from typing import Tuple
 
+# fmt: off
```

Comment:
Please do not use `fmt: off` here.

Reply:
I did it because otherwise imports take 40 lines, but I guess they can be collapsed.